tools/test.py

```diff
@@ -49,6 +49,7 @@
 from Queue import Queue, Empty
 
 logger = logging.getLogger('testrunner')
+is_skipped = re.compile(r"# SKIP\S+ (.+)", re.IGNORECASE)
```

Comment:
Can you use single quotes?  By the way, the "official" regular expression is `# SKIP\S*\s+(.*)`.